impress-org/give-double-the-donation
# Double the Donation settings have been reviewed

Resolves #2 

## Description
This PR addresses a handful of issues related to admin settings for the Double the Donation add-on. Specifically, the PR moves the settings into their own top-level tab, and updates form links.

## Affects
This PR affects the double the donation ServiceProvider (where setting pages are loaded) and replaces the SettingsTab class with the SettingsPage class to handle rendering the settings page content.

## Visuals
n/a

## Pre-review Checklist

## Testing Instructions
1. Check that the Double the Donation settings are in their own tab of the Settings page
2. Check that the registration form link uses a short url
3. Check that the docs URL exists
